### DISCUSSION TITLE
Ссылка из таблицы циклов на отфильтрованную простую таблицу

### DISCUSSION THREAD

**Discussion by gorbarov** [totumonline/totum-ru-issues-and-discussions#398] [Как это сделать?] [answered]
Точно помню, что где-то такой лайфхак видел. 

При том, что это не просто кнопка linkToTable, а она прям в табах внутри цикла появлялась. Но не могу вспомнить где я это видел.

**Comment by erpic-ru** [comment, marked as answer]
https://ru.docs.totum.online/tables#type-tab-in-cyclea

**Comment by gorbarov** [comment]
Спасибо @erpic-ru минут 20 искал и так и эдак )
